.phpcs.xml.dist

```diff
@@ -15,7 +15,6 @@
 
 		<!-- @TODO(P0): Fix these once WPCS is merged on develop and remove the exclusions. -->
 		<exclude name="PSR2.Methods.FunctionClosingBrace.SpacingBeforeClose" />
-		<exclude name="WordPress.Security.EscapeOutput.OutputNotEscaped" />
```

Comment:
How did you know to remove this?

Reply:
When we upgraded WPCS, it was giving multiple false positives for escaping. Recently while making this change - https://github.com/ampproject/amp-wp/pull/7816/files#diff-b898523222368e5082cb1fbed4a54d1062c1c6e8fcf7d859efb90a57a2aff1a6R472, I turned it off and found out everything is working as expected, so just decided to remove the exclusion.